### PR TITLE
GAT-1887: Testing release to preprod functionality gateway-api

### PR DESCRIPTION
**Pull request by reubensamuel**

### CODE REVIEW

**Comment by reubensamuel** [comment]
@loki-sinclair-hdruk @GK230 Loki, Geeta  could you please review and suggest if your changes are ok to go to preprod .This is to test the release process in preprod for gateway-api

File: src/resources/dataset/dataset.entity.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ export default class DatasetClass extends Entity {
 		transformedObject.dataset['@schema'] = {
 			type: `Dataset`,
 			version: `2.0.0`,
-			url: `https://raw.githubusercontent.com/HDRUK/schemata/master/schema/dataset/latest/dataset.schema.json`,
+			url: `https://raw.githubusercontent.com/HDRUK/schemata/master/schema/dataset/2.1/dataset.schema.json`,
```
Comment (loki-sinclair-hdruk) [inline src/resources/dataset/dataset.entity.js:53]: This isn't right, it should have changed to 2.1.0 ?

Comment (reubensamuel) [reply src/resources/dataset/dataset.entity.js:53]: uh oh could you please move the changes till release branch ?

Comment (loki-sinclair-hdruk) [reply src/resources/dataset/dataset.entity.js:53]: Will need @GK230 to comment. I know she updated, but I'm unsure if the change to the versioning of the schema was pushed yet or not

**Pull request merged**